bundles/org.openhab.binding.omatic/README.md

```diff
@@ -0,0 +1,219 @@
+# State O-Matic Binding
+The State O-Matic Binding will enable you to monitor electrical devices such as
+washing machine, dryer , dishwasher, electrical car charging and so on.
+The component is generic and any type of equipment can be monitored as long as
+it is possible to read its power consumption.
+
+Examples of usage: 
+- Monitor your electric car. Once the car has finished charging, you will get notified about the cost, consumption and time it took.
+- Monitor your dishwasher, dryer or washing machine. Once they are finished you will get notified about the cost, consumption and time it took.
+- Send info when the power consumption is above a certain limit for a device.
+
+<img src="https://raw.githubusercontent.com/seaside1/omatic/master/image/omatic.jpg" width=200 />
```

Comment:
You could include images in the `doc/` folder of this binding.

Reply:
Fixed in https://github.com/openhab/openhab-addons/pull/9286/commits/ebd0ae674be0e8107722de21ba936c75db3d6f20